app/graphql/resolvers/events_resolver.rb

```diff
@@ -18,6 +18,7 @@ def resolve(page: nil, limit: nil)
       if current_organization.clickhouse_events_store?
         Clickhouse::EventsRaw.where(organization_id: current_organization.id)
           .order(ingested_at: :desc)
+          .limit_by(1, "transaction_id, timestamp, external_subscription_id, code")
```

Comment:
i'm wondering if we can't take `events_enriched` instead as it should be "quick"? 🤔 

is it not a problem to not have `customer` here for instance?

Reply:
oh `delegate :customer, :customer_id, to: :subscription` actually it works with an event raw

Reply:
We could probably use events enriched but it would mean that nothing will be visible in the event list until the event is processed, and we would still need to deduplicate to handle the merging delay that can happen on clickhouse.